code/game/objects/structures/burrows.dm

```diff
@@ -446,8 +446,11 @@ percentage is a value in the range 0..1 that determines what portion of this mob
 		invisibility = 0
 		icon_state = "hole"
 		layer = OPEN_DOOR_LAYER //Basiclly we can see this over most things other then closed doors, and the like
+		plane = GAME_PLANE
 		name = "burrow"
 		desc = "Some sort of hole that leads inside a wall. It's full of hardened resin and secretions. Collapsing this would require some heavy digging tools"
```

Comment:
```suggestion
		desc = "Some sort of hole that leads inside a wall. It's full of hardened resin and secretions. Collapsing this would require some heavy digging tools."
```